packages/IMAPClient-Tests.package/ICContactTest.class/instance/setUp.st

```diff
@@ -0,0 +1,7 @@
+running
+setUp
+	
+	self contact: (ICContact new 
+					email: 'erika.musterfrau@rocks.de';
+					name: 'Erika Musterfrau';
+					yourself)
```

Comment:
At first I wanted to test also `newWith: with:` (class method of ICContact). Should we add this test? Otherwise we should instantiate the test object with `newWith: with:`

Reply:
I think it's fine if we just use `newWith: with:` here

Reply:
Ok